native/src/hooks/useEmbeddedOffers.ts

```diff
@@ -0,0 +1,56 @@
+import { useCallback } from 'react'
+
+import {
+  CategoryModel,
+  LoadSprungbrettJobReturn,
+  SPRUNGBRETT_OFFER,
+  loadSprungbrettJobs,
+  useLoadAsync,
+} from 'api-client'
+
+import { determineApiUrl } from '../utils/helpers'
+
+type EmbeddedOffers = {
+  sprungbrett?: Awaited<LoadSprungbrettJobReturn>
+}
+
+type EmbeddedOffersParams = {
+  category?: CategoryModel | null
+  cityCode: string
+  languageCode: string
+}
+
+export type EmbeddedOffersReturn = {
+  embeddedOffers: EmbeddedOffers | null
+  error: Error | null
+  loading: boolean
+  refresh: () => void
+}
+
+const useEmbeddedOffers = ({ category, cityCode, languageCode }: EmbeddedOffersParams): EmbeddedOffersReturn => {
```

Comment:
Optional: So far our naming that fetches stuff from the cms is including `load`.
```suggestion
const useLoadEmbeddedOffers = ({ category, cityCode, languageCode }: EmbeddedOffersParams): EmbeddedOffersReturn => {
```